README.md

```diff
@@ -10,11 +10,11 @@
 
 1. [Install `poetry`](https://python-poetry.org/docs/#installation)
    - [`pipx install poetry`](https://python-poetry.org/docs/#installing-with-pipx)
-2. Setup virtual environment and install dependencies.
+2. Set up virtual environment and install dependencies.
```

Comment:
What do you think about moving these README changes into a separate PR? Just in case we have to roll back the logging changes, it would be nice to not also have to roll back the README changes. Since this is an internal facing repo (the README isn't seen by users) I won't block the PR on this point.

Reply:
deal, fair point--will add to separate PR, ty

Reply:
spinoff PR is here: https://github.com/great-expectations/cloud/pull/81

Reply:
Thank you for separating! ✅  on spinoff.